### PR TITLE
Add QS for installing namespeced Helm Chart Repos

### PR DESCRIPTION
**Pull request by serenamarie125**
Adding a Quick Start for installing namespaced Helm Chart Repos.  This Quick Start will be linked to from the description of the Helm Chart Catalog.  Please see screenshots below:
Quick Start Card:
QS-01-Card

Quick Start Overview:
QS-02-Overview

Quick Start Step:
QS-03-Step1

Quick Start Verify:
QS-04-Step1-Verify

Quick Start Conclusion:
QS-05-Conclusion

### CODE REVIEW

**Comment by serenamarie125** [comment]
@debsmita1 @rohitkrai03 here is the PR for the new Quick Start

**Comment by serenamarie125** [comment]
@slemeur can you give a quick review?

**Comment by slemeur** [comment]
Tested. Looks great to me thanks!

**Review by spadgett — APPROVED** [review]
/approve

**Comment by jerolimov** [comment]
@serenamarie125 The code formatting looks strange after copy and paste the yaml:

image

On your screenshot it looks better but I would recommend to skip the empty lines in the yaml itself.

There is also no copy button for the yaml. Do we need an annotation for the code snippet so that it has a copy button?

File: quickstarts/install-helmchartrepo-ns.yaml [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+          1. Click on the [perspective switcher]{{highlight
+          qs-perspective-switcher}} at the top of the navigation, and select
+          **Developer**.
+
+          1. In the navigation menu, click [Add]{{highlight qs-nav-add}}.
+
+          1. On the **+Add** page, click **Helm Chart** in the **Developer
+          Catalog** tile. 
+
+          1. Do you see **My Helm Chart Repo** as an available catalog in the
```
Comment (debsmita1) [inline quickstarts/install-helmchartrepo-ns.yaml:88]:
```suggestion
          1. Do you see **My Helm Chart Repo** as an available chart repository in the
```

**Review by debsmita1 — CHANGES_REQUESTED** [review]

File: quickstarts/install-helmchartrepo-ns.yaml [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+              name: my-helm-chart-repo
+
+            spec:
+
+              connectionConfig:
+                url: 'https://raw.githubusercontent.com/Azure-Samples/helm-charts/master/docs'
+
+              name: My Helm Chart Repo
+
```
Comment (jerolimov) [inline quickstarts/install-helmchartrepo-ns.yaml:69]: Don't see a reason for these empty lines in the YAML example:

With this change it looks like this:

image

```suggestion
            apiVersion: helm.openshift.io/v1beta1
            kind: ProjectHelmChartRepository
            metadata:
              name: my-helm-chart-repo
            spec:
              connectionConfig:
                url: 'https://raw.githubusercontent.com/Azure-Samples/helm-charts/master/docs'
              name: My Helm Chart Repo
```

Did we add also an option to allow the user to copy the snippet? But I don't find any quickstart which uses this feature :shrug:

File: quickstarts/install-helmchartrepo-ns.yaml [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+        1. In the navigation menu, click [Search]{{highlight qs-nav-search}}.
+
+        1. In the project selector, select the project to populate with the new
+        Helm Charts.
+
+        1. Click the **Resources dropdown** menu and select
+        **ProjectHelmChartRepository**.
+
+        1. Click **Create Project Helm Chart Repository** to bring up the YAML
```
[the remark this answers is not in the document]
Comment (serenamarie125) [reply quickstarts/install-helmchartrepo-ns.yaml:49]: good find - I was going off what HelmChartRepository does.  Thanks for pointing this out!

File: quickstarts/install-helmchartrepo-ns.yaml (hunk 2 above)
Comment (serenamarie125) [reply quickstarts/install-helmchartrepo-ns.yaml]: Removed spaces.  We don't have snippets to date @jerolimov but maybe something to investigate for the future!

**Comment by debsmita1** [comment]
LGTM

**Review by jerolimov — APPROVED** [review]
/lgtm

**Comment by invincibleJai** [comment]
Propagate from epic:
/label docs-approved
/label px-approved

**Comment by debsmita1** [comment]
verified the QS on a cluster
/label qe-approved

**Pull request merged**